packages/fields/src/actions/duplicate-template-part.tsx

```diff
@@ -61,7 +63,7 @@ const duplicateTemplatePart: Action< TemplatePart > = {
 				onCreate={ onTemplatePartSuccess }
 				onError={ closeModal }
 				confirmLabel={ _x( 'Duplicate', 'action label' ) }
-				closeModal={ closeModal }
+				closeModal={ closeModal ?? ( () => void 0 ) }
```

Comment:
Why `void 0` and not `{}`

Reply:
Fixed with bb1f3668c2a43426e1899d74bd278989b787d024